podcasts/AppDelegate.swift

```diff
@@ -26,13 +26,17 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     private var backgroundSignOutListener: BackgroundSignOutListener?
 
+    var whatsNew: WhatsNew?
+
     // MARK: - App Lifecycle
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]? = nil) -> Bool {
         configureFirebase()
         TraceManager.shared.setup(handler: traceHandler)
         FileLog.shared.setup()
 
+        setupWhatsNew()
```

Comment:
This is here mainly because I need the number of the last opened version before the analytics logic overrides it.

But now that I'm look at it, maybe I just need to store something like `lastOpenedVersion`?

This way, the reason why this has to be called before `appLifecycleAnalytics.checkApplicationInstalledOrUpgraded` is not implicit. Any feedback is appreciated.

Reply:
Couldn't we just set the WhatsNew via the property which will be init'd before anything is called?

```
let whatsNew = WhatsNew()
```

Reply:
Not sure I follow, this is what `setupWhatsNew()` already does.

Reply:
Ah I think I misunderstood. I don't think we need to track another version number in the defaults. But I'm also not super keen on creating a long living object that isn't used that regularly.

Right now I don't have any better ideas and since this is lightweight we can leave it as it is.